lib/ansible/modules/monitoring/datadog_monitor.py

```diff
@@ -29,6 +29,13 @@
           - Your Datadog API key.
         required: true
         type: str
+    api_host:
+        description:
+          - The URL to the Datadog API. Default value is "https://api.datadoghq.com".
```

Comment:
`C()` formatting function ([see](https://docs.ansible.com/ansible/latest/dev_guide/developing_modules_documenting.html#linking-within-module-documentation)) should be used for values:
- `... Default value is C(https://api.datadoghq.com).`
- `... the C(DATADOG_HOST) environment variable`